finality-aleph/src/justification/requester.rs

```diff
@@ -16,6 +16,64 @@ use crate::{
     network, Metrics,
 };
 
+/// This structure is created for keeping and reporting status of BlockRequester
+pub struct LastRequest<B: BlockT> {
```

Comment:
Why the strange name? Shouldn't it be called something like `JustificationRequestStatus`?

Reply:
It was supposed to be only status of LastRequest, but `JustificationRequestStatus` sounds like a better name.